layouts/index.html

```diff
@@ -13,8 +13,8 @@
     id="main">
   <div class="usa-grid section-bg-parent">
     <div class="usa-width-whole billboard  section-bg">
-      <h1 class="billboard-display_text">The Government Innovation Platform</h1>
-      <h2 class="billboard-second_text">A platform by government developers, for government developers.</h1>
+      <h1 class="billboard-display_text">A Platform as a Service for government teams</h1>
```

Comment:
I'm still worried by `Platform as a Service`. Prefaced with `A` makes it seem like the reader should already know what this is.  I know it's defined in the next sentence but I think some people may stop here. I liked `Compliance as a Service` as more immediately accessible to govt; was that considered?